lib/entangler/executor/base.rb

```diff
@@ -56,6 +60,8 @@ def logger
         @logger ||= Entangler::Logger.new(log_outputs, verbose: @opts[:verbose])
       end
 
+      private
+
       def log_outputs
```

Comment:
This is only overridden in `Entangler::Executor::Master` where it is private. I don't see why it shouldn't be private on the superclass.